src/webots/nodes/utils/WbObjectDetection.cpp

```diff
@@ -358,25 +423,42 @@ bool WbObjectDetection::isContainedInFrustum(const WbAffinePlane *frustumPlanes)
   return true;
 }
 
-WbAffinePlane *WbObjectDetection::computeFrustumPlanes(const WbSolid *device, const double verticalFieldOfView,
-                                                       const double horizontalFieldOfView, const double maxRange) {
+WbAffinePlane *WbObjectDetection::computeFrustumPlanes(const WbSolid *device, double verticalFieldOfView,
+                                                       double horizontalFieldOfView, double maxRange, bool isPlanarProjection) {
```

Comment:
`const` keywords were removed in the master PR. I believe these can be restored to improve the code readability.